### PR TITLE
Document requirement of libzypp 12.25.8 for dup boo#1182629

### PR DESCRIPTION
**Pull request by lkocman**

### CODE REVIEW

**Review by nilxam — APPROVED** [review]
LGTM

File: xml/release-notes.xml [hunk 1]
```diff
@@ -305,11 +305,37 @@
...
+   You might still need to specify --allow-vendor-change during migration
+   in case you're using various OBS repositories signed with other keys.
+   
+   &opensuseleap; releases older than 15.2 do not contain this feature
+   as they have already reached EOL. All users are adviced to upgrade to 
+   &opensuseleap; 15.2 with latest updates prior upgrading to 15.3.
+   
+   Following parameters can be used as workaround for libzypp older than 12.25.8.
+   <command>zypper dup ... --allow-vendor-change --force-resolution ... </command>
    <para/>
```
Comment (KucharczykL) [inline xml/release-notes.xml:336]: The `/` should be at the beginning, not at the end.

File: xml/release-notes.xml [hunk 2]
```diff
@@ -305,11 +305,37 @@
...
-   <title>Upgrading from &opensuseleap; 15.2</title>
+   
+  <sect2 xml:id="sec.upgrade.152">
+   <title>Seamless upgrade from &opensuseleap; 15.2</title>
+   <!-- bnc#1182629 -->
+   <para>
+   &opensuseleap; 15.3 is newly built on top of binary rpms from &sles;
+   This change was introduced as part of Closing The Leap Gap (CtLG) effort
+   to bring &opensuseleap; and &sles; closer together.
+
```
Comment (KucharczykL) [inline xml/release-notes.xml:316]: Excess white space in `<para>` is removed so this is actually one big paragraph. You need to use `<para>` to make paragraphs.

File: xml/release-notes.xml [hunk 3]
```diff
@@ -305,11 +305,37 @@
...
+   You might still need to specify --allow-vendor-change during migration
+   in case you're using various OBS repositories signed with other keys.
+   
+   &opensuseleap; releases older than 15.2 do not contain this feature
+   as they have already reached EOL. All users are adviced to upgrade to 
+   &opensuseleap; 15.2 with latest updates prior upgrading to 15.3.
+   
+   Following parameters can be used as workaround for libzypp older than 12.25.8.
+   <command>zypper dup ... --allow-vendor-change --force-resolution ... </command>
```
Comment (KucharczykL) [inline xml/release-notes.xml:335]: What is the purpose of `...` here? The `dup` command does not need any other parameters, right?

**Pull request merged**